packages/opds-web-client/src/app.tsx

```diff
@@ -32,7 +32,7 @@ const OPDSCatalogRouterHandler = config => {
       };
       return (
         <PathForProvider pathFor={config.pathFor}>
-          <ActionsProvider>
+          <ActionsProvider proxyUrl={config.proxyUrl}>
```

Comment:
What's the proxyUrl?

Reply:
The proxyUrl is something that I didn't build but is an option in the `DataFetcher` which is instantiated in `ActionsContextProvider`, so I passed it through to there